### PR TITLE
Typos from marvin.cs.uidaho.edu: L

### PR DESCRIPTION
**Pull request by skangas**
This replaces PR #1961. I've added in several variations myself.

See: http://marvin.cs.uidaho.edu/misspell.html

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21157,14 +21157,26 @@ lables->labels
...
+lacker->lacquer
+lackered->lacquered
+lackeres->lacquers
+lackering->lacquering
+lackers->lacquers
+lackrimose->lachrymose
+lackrimosity->lachrymosity
+lackrimosly->lachrymosely
 laer->later, layer,
+laf->laugh
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21171]: `leaf`, `loaf`, `lad`, `lag`, `lac`, `kaf`/`kaph`, etc.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21157,14 +21157,26 @@ lables->labels
...
 lagacies->legacies
 lagacy->legacy
 laguage->language
 laguages->languages
 laguague->language
 laguagues->languages
 laiter->later
+lambast->lambaste
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21179]: It looks like `lambast`/`lambasts` are accepted variants of `lambaste`/`lambastes`, at least in the UK:
* [Cambridge](https://dictionary.cambridge.org/dictionary/english/lambaste)
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/definition/english/lambast)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/lambast)
* [Collins](https://www.collinsdictionary.com/dictionary/english/lambast)

Move to GB_US?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:21171]: Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:21179]: > Move to GB_US?

Yup. I also found this reference: https://grammarist.com/spelling/lambast-lambaste/

Comment (skangas) [reply codespell_lib/data/dictionary.txt:21179]: Another one of these:

```
E   error 'lambast' should be in aspell dictionaries (en_GB) for dictionary /home/skangas/wip/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
E   error 'lambasts' should be in aspell dictionaries (en_GB) for dictionary /home/skangas/wip/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
```

So I've added it to the list in #2657 instead, and removed it from this PR.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -22492,6 +22555,7 @@ loock->look, lock,
 loockdown->lockdown
 loocking->looking, locking,
 loockup->lookup, lockup,
+lood->lewd
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22558]: `blood`, `flood`, `mood`, `look`, etc.

Comment (skangas) [reply codespell_lib/data/dictionary.txt:22558]: Added.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**